tests/level_set/fe_surface_distance.cpp

```diff
@@ -0,0 +1,432 @@
+                const double phi = (*D_data)(ci);
+                const double err = (*E_data)(ci);
+                const double dV = (*W_data)(ci);
+                Pointer<CartesianPatchGeometry<NDIM> > patch_geom = patch->getPatchGeometry();
+                const double* const patch_dx = patch_geom->getDx();
+
+                if (std::abs(err) <= 4.0 * patch_dx[0])
+                {
+                    if (std::abs(phi) <= 1.2 * patch_dx[0])
+                    {
+                        E_interface += std::abs(err - phi) * dV;
+                        volume_near_interface += dV;
+                        num_interface_pts++;
+                    }
+                    (*E_data)(ci) = std::abs(err - phi);
+                }
+                else
+                {
+                    (*E_data)(ci) = 0.0;
+                }
+            }
+        }
+    }
+    num_interface_pts = SAMRAI_MPI::sumReduction(num_interface_pts);
+    E_interface = SAMRAI_MPI::sumReduction(E_interface);
+    volume_near_interface = SAMRAI_MPI::sumReduction(volume_near_interface);
+
+    return;
+} // calculate_error_near_band
+
+/*******************************************************************************
+ * For each run, the input filename and restart information (if needed) must   *
+ * be given on the command line.  For non-restarted case, command line is:     *
+ *                                                                             *
+ *    executable <input file name>                                             *
+ *                                                                             *
+ * For restarted run, command line is:                                         *
+ *                                                                             *
+ *    executable <input file name> <restart directory> <restart number>        *
+ *                                                                             *
+ *******************************************************************************/
+int
+main(int argc, char* argv[])
+{
+    // Initialize libMesh, PETSc, MPI, and SAMRAI.
+    LibMeshInit init(argc, argv);
+    SAMRAI_MPI::setCommunicator(PETSC_COMM_WORLD);
+    SAMRAI_MPI::setCallAbortInSerialInsteadOfExit();
+    SAMRAIManager::startup();
+
+    { // cleanup dynamically allocated objects prior to shutdown
+
+        // Parse command line options, set some standard options from the input
+        // file, initialize the restart database (if this is a restarted run),
+        // and enable file logging.
+        Pointer<AppInitializer> app_initializer = new AppInitializer(argc, argv, "FESurfaceDistance.log");
+        Pointer<Database> input_db = app_initializer->getInputDatabase();
+
+        // Get various standard options set in the input file.
+        const bool dump_viz_data = app_initializer->dumpVizData();
+        const int viz_dump_interval = app_initializer->getVizDumpInterval();
```

Comment:
viz_dump_interval, uses_visit, uses_exodus, dump_timer_data, and timer_dump_interval are all unused. Would you please remove them?